### ISSUE TITLE
Dynamic Version

### ISSUE THREAD

**Issue by rac021** [tacoss/raml-mockup#17]
Hi,

I have the following  **baseUri** in a raml file :

```
   baseUri: http://192.168.56.1:8000/{version}
```

After running  raml-mockup  command and calling the resource : 

```
     http://localhost:5000/v1/planes
```

this gives me the following error :

 **{"error":"Missing resource for /planes"}**
# But ( **without specifying version** )

```
     http://localhost:5000/planes
```

this works fine for me.

Is it an issue ?

Thank's

**Comment by pateketrueke** [comment]
Yeah, all routes should have been composed correctly, even taking into account the base uri and such.

Let me see what can I do, thanks for your feedback!
